runtime/fail_nat.c

```diff
@@ -108,11 +110,10 @@ void caml_raise_with_args(value tag, int nargs, value args[])
 {
   CAMLparam1 (tag);
   CAMLxparamN (args, nargs);
-  value bucket;
+  CAMLlocal1 (bucket);
```

Comment:
Revert to trunk.

Reply:
Fixed in https://github.com/ocaml-multicore/ocaml/commit/7131fef3bc9b8405a4c7b47a82717fcbed516a01